andersoncd/plot_utils.py

```diff
@@ -26,3 +27,17 @@ def configure_plt():
     sns.set_palette('colorblind')
     sns.set_context("poster")
     sns.set_style("ticks")
+
+
+def plot_legend_apart(ax, figname, ncol=None):
```

Comment:
maybe make this private as Im unsure pdfcrop is installed on every system and it's for our own use only ?